tests/internal_bench/bench_hw.py

```diff
@@ -0,0 +1,12 @@
+import utime
```

Comment:
I suggest using just `import time`.  We are trying to move away from u-modules, and `utime` is rarely used now.